process_report/process_report.py

```diff
@@ -190,6 +221,15 @@ def main():
         alias_file = fetch_s3_alias_file()
     alias_dict = load_alias(alias_file)
 
+    if args.prepay_debits:
+        prepay_debits_filepath = args.prepay_debits
+    else:
+        prepay_debits_filepath = fetch_s3_prepay_debits()
```

Comment:
I should have spoken earlier about this, but we have too many of these `fetch_something_from_s3()` type methods that take no argument. We should factor this out and then re-use the methods. 
```python
def fetch_from_s3(local_name, s3_path):
    # do the thing
```

this doesn't have to be part of this PR.